src/extension/background-script/mnemonic/index.ts

```diff
@@ -3,7 +3,12 @@ import { HDKey } from "@scure/bip32";
 import * as bip39 from "@scure/bip39";
 import Hex from "crypto-js/enc-hex";
 import sha256 from "crypto-js/sha256";
+import { SLIP77Factory } from "slip77";
 
+import * as tinysecp from "../liquid/secp256k1";
+
+export const LIQUID_DERIVATION_PATH = "m/84'/1776'/0'/0/0";
```

Comment:
Can these constants be moved to `src/extension/background-script/liquid/index.ts`?

Reply:
Fixed in https://github.com/getAlby/lightning-browser-extension/pull/2583